build.gradle.kts

```diff
@@ -52,6 +52,7 @@ val compileJavaJvmArgs by extra((project.properties["compileJavaJvmArgs"] as Str
 val shouldRunAllCheckers by extra(isCI || project.hasProperty("runAllCheckers"))
 
 tasks.register<UpdateEapBuildNumber>("updateEapBuildNumber")
+tasks.register<UpdateIntellijStableVersions>("updateIntellijStableVersions")
```

Comment:
Actually... haven't checked the implementation yet, but I'm curious if it won't be easier to just merge these tasks into a single `updateIntellijVersions`, WDYT?

Reply:
Which would update both EAP and stable versions, depending on what's available

Reply:
Sure, why not!